flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -539,6 +524,79 @@ class FlxTilemapExt extends FlxTilemap
 	}
 	
 	/**
+	 * Sets the slope arrays, which define which tiles are treated as slopes of 22.5 degrees.
+	 * 
+	 * @param 	HighSlopes 	An array containing the numbers of the tiles to be treated as high slope tiles with a slope on the left.
+	 * @param 	LowSlopes	An array containing the numbers of the tiles to be treated as low slope tiles with a slope on the right.
+	 */
+	public function setSlopes22(?HighSlopes:Array<Int>, ?LowSlopes:Array<Int>) 
```

Comment:
"steep" and "gentle" are valid function names:
`setSteep(HighSlopes, LowSlopes)` / `setGentle(HighSlopes, LowSlopes)`
or
`setSteep(ThickSlopes, ThinSlopes)` / `setGentle(ThickSlopes, ThinSlopes)`